inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -62,6 +64,8 @@ protected function set_schema() {
 			url              varchar(2000)       NOT NULL default '',
 			css              longtext                     default NULL,
 			hash             varchar(32)                  default '',
+			error_code       varchar(32)                NULL default NULL,
```

Comment:
```suggestion
			error_code       varchar(32)             NULL default NULL,
```

remove three spaces to be correctly aligned (small and silly change I know :D )